src/pages/campaigns/create.tsx

```
@@ -0,0 +1,539 @@
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="startingLevel"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel className="text-white">Starting Level</FormLabel>
+                  <Select
+                    onValueChange={(val) => {
+                      field.onChange(parseInt(val));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)